### DISCUSSION TITLE
amazon.de - delivered - no count

### DISCUSSION THREAD

**Discussion by jeanjacquesjj** [moralmunky/Home-Assistant-Mail-And-Packages#994] [General]
Hello,

I just started with Mail & Packages and saw that my Amazon "delivered" sensor is not increasing. In my case 2 Amazon packages delivered in Germany and 2 mails received today in German. 

I'm absolutely not capable of reading code, but I saw an issues in the log-file:
- the title of the delivery mail is "Zugestellt: deine Amazon.de-Bestellung", search pattern seems to be: SUBJECT "Geliefert:" SINCE 11-Nov-2024 ...
So it says: _Amazon delivered email(s) found: 0_

It added "Zugestellt:", in line 102 of const.py and the sensor wet to "2" - perfect !

Perhaps this helps, but once again, I'm far from being a developer !
Thank you so much for your work !
JJ

**Comment by firstof9** [comment]
Thanks I created an issue #995  regarding this to add that to the subject search. :beers:
